### PR TITLE
Fix agenda slide and agendaIsProjected function.

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@ostcar There is still a bug in showing Agenda Slide. 
If you activate a agenda slide and then reload the projector slide all agenda items disappear ($scope.items is empty). How can I fill 'items' outside of $scope.$watch in SlideItemListCtrl? Just printing the same line for $scope.items = AgendaTree.getFlatTree(..) does not work.

File: openslides/agenda/static/js/agenda/agenda.js [hunk 1]
```diff
@@ -265,10 +265,10 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
             var projector = Projector.get(1);
             if (typeof projector === 'undefined') return false;
             var self = this;
-            return _.findIndex(projector.elements, function(element) {
+            var predicate = function (element) {
                 return element.name == 'agenda/item-list'
```
Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/agenda.js]: Add `;` here.

Comment (emanuelschuetze) [inline openslides/agenda/static/js/agenda/agenda.js]: Fixed

**Comment by emanuelschuetze** [comment]
@ostcar Please test this branch. Then you will see that the order of AgendaTree is still wrong (in list view, order view and agenda slide). Something is still broken...

File: openslides/agenda/static/js/agenda/agenda.js [hunk 2]
```diff
@@ -49,6 +49,68 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
     }
 ])
 
+.factory('AgendaTree', [
+    function () {
+        return {
+            getTree: function (items) {
+                // TODO: sortieren nach weight???
```
Comment (ostcar) [inline openslides/agenda/static/js/agenda/agenda.js]: Please change this line into:
```javascript
                // Sort items after there weight
                items.sort(function(itemA, itemB) {
                    return itemA.weight - itemB.weight;
                });
```

**Comment by emanuelschuetze** [comment]
All done. Thx to @ostcar!
Rebased and ready to merge.

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**